client/src/JobPaginatorState.js

```diff
@@ -0,0 +1,129 @@
+import AjaxHelper from "./AjaxHelper";
+
+const confirmSavedMagicLabels = (count) => {
+    return window.confirm("You will be saving " + count + " unreviewed, auto-generated label(s). Are you sure?");
+};
+
+const findNewPagePosition = (page, list) => {
+    const foundIndex = list.findIndex((listItem) => {
+        return listItem.filename >= page;
+    });
+    return foundIndex === -1 ? list.length : foundIndex;
+};
+
+const getJob = (category, job) => {
+    const ajax = AjaxHelper.getInstance();
+    return new Promise((resolve /*, reject*/) =>
+        ajax.getJSON(ajax.getJobUrl(category, job), null, (data) => {
+            resolve(data);
+        })
+    );
+};
+
+const getStatus = (category, job) => {
+    const ajax = AjaxHelper.getInstance();
+    return new Promise((resolve) =>
+        ajax.getJSON(ajax.getJobUrl(category, job, "/status"), null, (data) => {
+            resolve(data);
+        })
+    );
+};
+
+const deleteImage = (imageUrl, success, error) => {
```

Comment:
I just tested and found that the "delete image" feature doesn't work on this branch for some reason. I think that imageUrl is not getting constructed correctly, because when I look at my network console, I see DELETE requests going to URLs like: `http://localhost:9000/api/ingest/dime/job2/undefined/6`. Obviously, if you want to test this, you'll want to make a copy of your example job to test with so that you don't lose your original test data.

Reply:
Made change.